server/src/header.js

```diff
@@ -0,0 +1,33 @@
+const React = require("react");
+const PropTypes = require("prop-types");
+const { Localized } = require("fluent-react/compat");
+
+exports.Header = class Header extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if (!this.props.hasLogo) {
+      return <header className="header-panel default-color-scheme">
+        {this.props.children}
+      </header>;
```

Comment:
I'm usually a fan of early returns, but in this case `const logo = maybe ? <blah> :  null;` then `{ logo }` reads better.  It's also unclear if no logo means no `alt-actions` wrap, or that's just missing.